student.rb

```diff
@@ -0,0 +1,16 @@
+require_relative 'person'
+
+class Student < Person
+  def initialize(classroom)
```

Comment:
- Let's also update the constructor to `def initialize(age, classroom, name = "Unknown", parent_permission = true)` so it could extend its parent constructor in our case the